cg/server/api.py

```diff
@@ -299,8 +304,8 @@ def flowcells() -> Any:
         flow_cell_statuses=[request.args.get("status")],
         flow_cell_id_enquiry=request.args.get("enquiry"),
     )
-    parsed_flow_cells: List[dict] = [flow_cell.to_dict() for flow_cell in flow_cells.limit(50)]
-    return jsonify(flowcells=parsed_flow_cells, total=flow_cells.count())
+    parsed_flow_cells: List[dict] = [flow_cell.to_dict() for flow_cell in flow_cells[:50]]
```

Comment:
Just to keep the convention I have been using in my comments above, `Dict` instead of `dict`.
```suggestion
    parsed_flow_cells: List[Dict] = [flow_cell.to_dict() for flow_cell in flow_cells[:50]]
```
Also the 50 literal here